app/frontend/src/submissions/components/SubmissionForm/MethodOfDrilling.vue

```diff
@@ -57,8 +57,8 @@
             <b-form-radio-group v-model="wellOrientationInput"
                                 stacked
                                 name="wellOrientationRadio">
-              <b-form-radio value="VERTICAL">Vertical</b-form-radio>
-              <b-form-radio value="HORIZONTAL">Horizontal</b-form-radio>
+              <b-form-radio value="True">Vertical</b-form-radio>
```

Comment:
I though we had decided that using the codes makes more sense than using the boolean values?